### ISSUE TITLE
When I hit options button nothing happens (master build)

### ISSUE THREAD

**Issue by khthe8th** [AHAAAAAAA/PokemonGo-Map#2680]
I'm on the master build and all of a sudden when I hit the options button the menu no longer opens. It changes the URL to http://localhost:5000/#nav but the menu doesn't slide open like it used to. I don't think I changed anything so I'm not sure why this is happening all of a sudden, it was working fine a couple hours ago

**Comment by andrewda** [comment]
Open your developer console. Are there any issues?

**Comment by Pringles91** [comment]
This whole thing was working fine 2 days ago, now i either dont see any pokemon, or when i do there is 30 seconds left to go get them. pretty bummed :(

**Comment by andrewda** [comment]
@Pringles91 completely unrelated to this issue. There are many issues open regarding your issue (spoiler - Niantic added request throttling) but if you still need help, checkout the #help channel on discord: https://discord.gg/sSYhW

**Comment by khthe8th** [comment]
dumb question but how do i open the dev console? Before the options menu stopped working I turned on scan locations so I can see that I am still constantly scanning and updating locations/pokemon

**Comment by andrewda** [comment]
If you're using Google Chrome:

![screen shot 2016-07-29 at 7 23 55 pm](https://cloud.githubusercontent.com/assets/10191084/17267601/38790292-55c2-11e6-967c-e5317e16e1e5.png)

**Comment by khthe8th** [comment]
![image](https://cloud.githubusercontent.com/assets/5308389/17267622/bd68a91c-55c2-11e6-8454-8177cc44f414.png)

**Comment by andrewda** [comment]
You're blocking a lot of requests to the Google Maps API for some reason (maybe some Chrome extension?). Also looks like there's some JavaScript issue that I'll look into.

**Comment by khthe8th** [comment]
hmm that's weird, any idea what would cause that? Like I said I didn't change anything in chrome, it just suddenly stopped working

**Comment by andrewda** [comment]
@khthe8th did you recently update the app or did it literally stop working out of the blue?

**Comment by khthe8th** [comment]
It worked, then I was trying to get the dev build of the map to work but it kept failing on npm install so I went back to my master build and options stopped working. While messing around with dev build I didn't touch anything in my master build folder though, but maybe trying to get dev build to work changed something on local host?

When I run the dev build everything works except the map doesn't load, stays grey like a lot of opened issues are talking about. The options button in the dev build works though

**Comment by khthe8th** [comment]
I just refreshed my server and re-opened local host and all those other errors about blocked API requests are no longer in my dev console. Only one left is the top one about addEventListener. Options button still doesn't work.

**Comment by Hoob32** [comment]
Any update on this? Got the same issue, tried the dev build for beehive and got that somewhat working and came back to try and use the 2.1.0 the way I normally do (could open options before downloading dev build etc) and now I can't open the options menu. Got the same error in the dev tools as above.

**Comment by Pringles91** [comment]
Any update? my options button also does not work

**Comment by Legoboy725** [comment]
I had to turn off my adblocker to get the options menu to open up again.  I use uBlock Origin on Chrome.

**Comment by Pringles91** [comment]
i dont have any ad blockers. i used it one night, changed nothing, tried using it the next day and bam, does the same thing khthe8th said, soon as you click options, all it does it change the url to http://localhost:5000/#nav

**Comment by khthe8th** [comment]
Im using uBlock Origin too and I've disabled it on my localhost webpage but it still does not work

**Comment by Hoob32** [comment]
Yeah I've disabled all my extensions including adblock and still wont work, only way I can get the options menu is when I ngrok to my phone and works fine there ;/

**Comment by Hoob32** [comment]
Interesting, using localhost:5000 results in map working but no options, but using 127.0.0.1:5000 results in map working and options working like normal.

**Comment by khthe8th** [comment]
using  127.0.0.1:5000 the options menu still does not work for me
